fields/advanced_file/config_template.php

```diff
@@ -1,61 +1,70 @@
-<div class="caldera-config-group">
```

Comment:
@pascalknecht  I see what you are doing with the role attribute. The problem here is that this patch is about adding 1 setting. By modifying every setting its going to make merging it with changes in #2766 

Please revert changes to existing fields. This is a legit #a11y issue that we will not repeat when we refactor this UI.

Reply:
Hold on, I'm wrong. Looks like this diff is beacuse of auto-formatting. 

Either way we need it to just be adding that 1 setting

Reply:
@Shelob9 Yes they are indeed only auto formatting changes. Tried to do that but it always inserts those formats.